app/Kconfig

```diff
@@ -7,4 +7,21 @@ config APP_DATA_LOGGING_SYNC_INTERVAL
 		This configures the interval at which the logging system flushes everything to disk
 		and hence determines the maximum duration of data lost on power cuts.
 
+choice
+	prompt "IMU"
+	help
+		This configures Eurus Flight Controller primary IMU.
+
+config APP_DATA_USE_MPU6050
```

Comment:
Change these to match the name of the choice and add the IMU name after, for suggested example: `APP_PRIMARY_IMU_MPU6050`. This would be in-line with how Kconfig options are usually done.